troubleshoot.md

```diff
@@ -0,0 +1,24 @@
+# Troubleshooting
+
+- [Troubleshooting](#troubleshooting)
+  - [Celestia-app](#celestia-app)
+    - [Validator Node setup](#validator-node-setup)
+
+## Celestia-app
+
+### Validator Node setup
+If you get an error such as 
+
+```Error: <keyname>: key not found```,
+
+this means your key, the field referenced by the `--from` option, does not exist.
+
+You can fix this by adding your key manually to the keyring via:
+
+```
+celestia-appd keys add --recover <keyname>
+```
```

Comment:
might also want to mention using a different --keyring-backend, the script mentioned using the test backend, but this might not be the best default for people setting up a devnet node, so we might need to change that too

Reply:
I agree.